project/TestParallelization.scala

```diff
@@ -0,0 +1,106 @@
+import sbt.Keys._
+import sbt._
+
+object TestParallelization {
+
+  lazy val testGroupingInDifferentJVMSettings = Seq(
```

Comment:
for naive SBT users like me, it would be good to add inline docs explaining what each of these chunks of code does

Reply:
I will add some comments, or do you prefer some other form (i.e. using two objects for the two settings group and comment the object)